### PR TITLE
Update to stack tools + Fixes

### PR DESCRIPTION
**Pull request by ardumont**
- [X] Migrate travis-ci to stack
- [X] Migrate Makefile to stack (+ new target install to deploy stack locally)
- [X] Fix tests
- [X] Fix nmcli api call

### CODE REVIEW

File: Network/Nmcli.hs [hunk 1]
```diff
@@ -27,7 +27,7 @@ scanCmd = Scan "nmcli --terse --fields ssid,signal dev wifi"
 
 -- | Command to list the wifi the computer can currently auto connect to
 knownCmd :: Command
-knownCmd = Scan "nmcli --terse --fields name con list"
```
Comment (massyl) [inline Network/Nmcli.hs]: did you test this change with Mint or Ubuntu?

Comment (ardumont) [inline Network/Nmcli.hs]: With debian :D

Comment (massyl) [inline Network/Nmcli.hs]: can you check if still working on Mint or Ubuntu please?

Comment (ardumont) [inline Network/Nmcli.hs]: Unfortunately, I have no more mint nor ubuntu.
I can check on nixos though (but not right now).

Comment (ardumont) [inline Network/Nmcli.hs]: Ex: 

``` sh
Scanned wifi: 

- xyz
- abc
- 123
- FreeWifi_secure

Auto-connect wifi: 

- 123
- abc
- xyz

Connection to wifi '123'
Connection successfully activated (D-Bus active path: /org/freedesktop/NetworkManager/ActiveConnection/9)
```

Comment (ardumont) [inline Network/Nmcli.hs]: You can check on your side (without merging) if it's working with this:

```sh
nmcli --terse --fields name con
```
vs the original:

```sh
nmcli --terse --fields name con list
```

Comment (massyl) [inline Network/Nmcli.hs]: It's working :-)

**Pull request merged**